lua/lvim/core/nvimtree.lua

```diff
@@ -67,26 +65,46 @@ function M.config()
         relativenumber = false,
         signcolumn = "yes",
       },
+      renderer = {
+        indent_markers = {
+          enable = false,
+          icons = {
+            corner = "└ ",
+            edge = "│ ",
+            none = "  ",
+          },
+        },
+        icons = {
+          webdev_colors = true,
+        },
+      },
       filters = {
         dotfiles = false,
         custom = { "node_modules", "\\.cache" },
+        exclude = {},
       },
       trash = {
         cmd = "trash",
         require_confirm = true,
       },
       actions = {
```

Comment:
```suggestion
      },
      log = {
        enable = false,
        truncate = false,
        types = {
          all = false,
          config = false,
          copy_paste = false,
          diagnostics = false,
          git = false,
          profile = false,
        },
      },
      actions = {
```
this looks useful, wdyt?

ref: https://github.com/kyazdani42/nvim-tree.lua/blob/fd2332a33f0fc7c8545cd1c358ad032876318a43/doc/nvim-tree-lua.txt#L186-L197